Automattic/tumblr-theme-garden
# Increase the wait time before searching

### Summary

The search time before submitting a request is quite short, the search experience is a bit quirky due to that, so increasing this time helps to improve the search for slow typers.

### Solution

This PR increases the wait time before searching:
- before: 500 ms
- after: 1000 ms

### Testing steps

- `git checkout` this branch
- `npm install`
- `npm run build`
- `composer install`
- now install and activate this plugin
- navigate to `/wp-admin`
- go to `Appearance > Tumblr Themes`
- search on the filter bar, ensure the wait time before actually showing results is reasonable

### Reviewing tips

🧁 Request a review to team CupcakeLabs to assign a random team member.

🔄 Feel free to ping the team name again to "re-roll" or add another team member.

🚀 For now there is no requirement on getting review approvals before merging.
